### PR TITLE
Update Chinese translations

### PR DESCRIPTION
**Pull request by Disappear9**

### CODE REVIEW

File: gui/public/i18n/zh-Hans/translation.ftl [hunk 1]
```diff
@@ -154,29 +154,29 @@ body_assignment_menu-unassign_tracker = 取消分配追踪器
...
+tracker_selection_menu-WAIST = { -tracker_selection-part } 腰部？
+tracker_selection_menu-HIP = { -tracker_selection-part } 髋部？
+tracker_selection_menu-LEFT_SHOULDER = { -tracker_selection-part } 左肩？
+tracker_selection_menu-LEFT_UPPER_ARM = { -tracker_selection-part } 左大臂？
+tracker_selection_menu-LEFT_LOWER_ARM = { -tracker_selection-part } 左小臂？
+tracker_selection_menu-LEFT_HAND = { -tracker_selection-part } 左手？
+tracker_selection_menu-LEFT_UPPER_LEG = { -tracker_selection-part } 左大腿？
+tracker_selection_menu-LEFT_LOWER_LEG = { -tracker_selection-part } 左小腿？
+tracker_selection_menu-LEFT_FOOT = { -tracker_selection-part } 左脚？
+tracker_selection_menu-LEFT_CONTROLLER = { -tracker_selection-part } 左控制器？
```
Comment (ImUrX) [inline gui/public/i18n/zh-Hans/translation.ftl:179]: You can remove the space after the ``}`` if you want :p

File: gui/public/i18n/zh-Hans/translation.ftl [hunk 2]
```diff
@@ -423,11 +420,7 @@ onboarding-connect_tracker-connection_status-handshake = 已连接到服务器
 # English in this case only has 2 plural rules, which are "one" and "other",
 # we use 0 in an explicit way because there is no plural rule in english for 0, so we directly say
 # if $amount is 0 then we say "No trackers connected"
-onboarding-connect_tracker-connected_trackers = { $amount ->
-    [0] No trackers
-    [one] 1 tracker
-    *[other] { $amount } trackers
-} connected
+onboarding-connect_tracker-connected_trackers = { $amount } 个追踪器已连接
```
Comment (ImUrX) [inline gui/public/i18n/zh-Hans/translation.ftl:423]: Don't you want to say a different message when there are no trackers connected?

File: gui/public/i18n/zh-Hans/translation.ftl [hunk 3]
```diff
@@ -260,29 +260,26 @@ settings-general-fk_settings-skeleton_settings-description = 打开或关闭骨
...
-settings-general-gesture_control-quickResetEnabled = Enable tap to quick reset
-settings-general-gesture_control-quickResetDelay = Quick reset delay
-settings-general-gesture_control-quickResetTaps = Taps for quick reset
-settings-general-gesture_control-resetEnabled = Enable tap to reset
-settings-general-gesture_control-resetDelay = Reset delay
-settings-general-gesture_control-resetTaps = Taps for reset
-settings-general-gesture_control-mountingResetEnabled = Enable tap to reset mounting
-settings-general-gesture_control-mountingResetDelay = Mounting reset delay
-settings-general-gesture_control-mountingResetTaps = Taps for mounting reset
+settings-general-gesture_control-taps = { $amount } 次敲击
```
Comment (ImUrX) [inline gui/public/i18n/zh-Hans/translation.ftl:273]:
```suggestion
settings-general-gesture_control-taps = { $amount ->
    *[other] { $amount }次敲击
}
```

File: gui/public/i18n/zh-Hans/translation.ftl [hunk 4]
```diff
@@ -437,10 +430,7 @@ onboarding-assign_trackers-description = 让我们选择哪个追踪器在哪里
 # Look at translation of onboarding-connect_tracker-connected_trackers on how to use plurals
 # $assigned (Number) - Trackers that have been assigned a body part
 # $trackers (Number) - Trackers connected to the server
-onboarding-assign_trackers-assigned = { $assigned } of { $trackers ->
-    [one] 1 tracker
-    *[other] { $trackers } trackers
-} assigned
+onboarding-assign_trackers-assigned = { $assigned } / { $trackers } 个追踪器已分配
```
Comment (ImUrX) [inline gui/public/i18n/zh-Hans/translation.ftl:433]:
```suggestion
onboarding-assign_trackers-assigned = { $assigned }/{ $trackers } 个追踪器已分配
```

File: gui/public/i18n/zh-Hans/translation.ftl [hunk 5]
```diff
@@ -502,10 +492,7 @@ onboarding-automatic_proportions-recording-steps-3 = 向右扭转上身，然后
 onboarding-automatic_proportions-recording-steps-4 = 持续摆动身体，直到计时器结束。
 onboarding-automatic_proportions-recording-processing = 正在处理结果
 # $time (Number) - Seconds left for the automatic calibration recording to finish (max 15)
-onboarding-automatic_proportions-recording-timer = { $time ->
-    [one] 1 second left
-    *[other] { $time } seconds left
-}
+onboarding-automatic_proportions-recording-timer =  剩余 { $time } 秒
```
Comment (ImUrX) [inline gui/public/i18n/zh-Hans/translation.ftl:495]:
```suggestion
onboarding-automatic_proportions-recording-timer =  剩余{ $time }秒
```

**Review by ImUrX — CHANGES_REQUESTED** [review]

**Review by ImUrX — APPROVED** [review]

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**